src/unix/fs.c

```diff
@@ -125,6 +125,10 @@
   }                                                                           \
   while (0)
 
+#if defined(__MVS__)
+#define readlink(...) os390_readlink(__VA_ARGS__)
```

Comment:
Shouldn't `os390_readlink()` be defined in a header somewhere?

Reply:
This macro definition is only meant for readlink in this file.
I will remove this and use the ifdef below where readlink is actually called.

Reply:
What I mean is, shouldn't there be a function prototype for `os390_readlink()` in a header?  The function is in another source file.

Reply:
Understood. I will put it in os390-syscalls.h